### ISSUE TITLE
cssnano option orderedValues conflicts with css-modules imports

### ISSUE THREAD

**Issue by wereHamster** [webpack-contrib/css-loader#316]
The cssnano optimization **orderedValues** uses the `postcss-ordered-values` module to process the CSS AST and normalizes `border: X` declarations. It tries to parse the right-hand of `border:` and extract the width, style and color. It parse the width using `postcss-value-parser`. However, when the `border:` declaration contains values imported from another file, cssnano sees strings like `___CSS_LOADER_IMPORT___7___` instead of the pixel values. So `postcss-value-parser` fails to parse the width and drops it form the output.

In my case, I have a this CSS code:

``` css
@value selectInputBorderWidth, selectInputBorderColor from "./variables.css";
.Select-control {
    border: selectInputBorderWidth solid selectInputBorderColor;
}
```

and the output is:

``` css
.Select-control {
    border: solid #666666;
}
```

cssnano does not (and can not, at that stage in the webpack loader pipeline) see the faithful representation of the CSS code with all values and imports fully resolved. I can try to disable **orderedValues** (which solves this particular bug in my particular instance) but I do wonder which other optimizations are unsafe. The safest option is to remove cssnano completely from css-loader.

TLDR: remove cssnano, it can not work reliably.

**Comment by ben-eb** [comment]
Hi @wereHamster,

Would you be able to provide a reduced test case so that I can reproduce? It might be worth looking into a compatibility mode for this use case; we already return early when the value contains any CSS variables, so this might not be too difficult to implement.

Thanks!

**Comment by wereHamster** [comment]
The first code snippet should be enough of an example. Or do you need more? Maybe it works now with the latest version of the relevant modules. I haven't tested our code since I raised this issue... in.. August last year.

**Comment by ben-eb** [comment]
Just the configuration would be helpful. I'm not a heavy webpack user, nor do I watch this repository, but if I can help resolve this on my end then that'll be beneficial to other users too.

**Comment by wereHamster** [comment]
@ben-eb 

```
git clone https://gist.github.com/wereHamster/b0d1a46e6ecd438b95346cf75c4717a4 css-loader-316
cd css-loader-316
npm install
npm run build
cat build/style.css
```

And observe that the border width from the original file (main.css) disappeared.

**Comment by ben-eb** [comment]
@wereHamster Thanks very much for doing that. I've pushed a new version of postcss-ordered-values that now resolves this issue. 👍

**Comment by evilebottnawi** [comment]
Solved in `cssnano`
